doc/sphinx/Pacemaker_Development/c.rst

```diff
@@ -880,6 +880,30 @@ __________________________________
 * The convenience macros ``pcmk__plural_s()`` and ``pcmk__plural_alt()`` are
   handy when logging a word that may be singular or plural.
 
+Log Levels
+__________
+
+When to use each log level:
+
+* **critical:** fatal error (usually something that would make a daemon exit)
+* **error:** failure of something that affects the cluster (such as a resource
+  action, fencing action, etc.) or daemon operation
+* **warning:** minor, potential, or recoverable failures (such as something
+  only affecting a daemon client, or invalid configuration that can be left to
+  default)
+* **notice:** important successful events (such as a node joining or leaving,
+  resource action results, or configuration changes)
+* **info:** events that would be helpful with troubleshooting (such as status
+  section updates, elections)
```

Comment:
Probably change this to `section updates or elections`.

Reply:
Good catch, updated